ocp_resources/virtual_machine.py

```diff
@@ -89,11 +89,13 @@ def restart(self, timeout=TIMEOUT_4MINUTES, wait=False):
             self.vmi.virt_launcher_pod.wait_deleted()
             return self.vmi.wait_until_running(timeout=timeout, stop_status="dummy")
 
-    def stop(self, timeout=TIMEOUT_4MINUTES, wait=False):
+    def stop(
+        self, timeout=TIMEOUT_4MINUTES, vmi_delete_timeout=TIMEOUT_4MINUTES, wait=False
+    ):
         self.api_request(method="PUT", action="stop")
         if wait:
             self.wait_for_status(timeout=timeout, status=None)
-            return self.vmi.wait_deleted()
+            return self.vmi.wait_deleted(timeout=vmi_delete_timeout)
```

Comment:
I don't know. It seems reasonable to me that a VMI will take a while to delete, and less reasonable for a VM.
We can determine it's reasonable, but I don't want to miss the time a VM reaches timeout (missing on an opportunity to determine this), because VM delete_timeout was manipulated to support VMI deletion

Reply:
NVM, we can manipulate VMI delete_timeout. Went over my head :)